src/repl/repl_annotations.cpp

```
@@ -203,17 +203,38 @@ void show_diff(const std::string& old_text, const std::string& new_text, std::os
   }
 }
 
+/// Extract decline reason from user answer (e.g. "n, use sed" → "use sed").
+/// Returns empty string if no reason given (bare "n" or "no").
+static std::string extract_decline_reason(const std::string& answer) {
+  // Strip leading "n"/"no" and optional separator (, or space)
+  std::string rest;
+  if (answer.size() > 1 && (answer[0] == 'n' || answer[0] == 'N')) {
+    if (answer.substr(0, 2) == "no" || answer.substr(0, 2) == "No") {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)